_dashboards/visualizations.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: default
+title: Visualizations
+nav_order: 40
+---
+
+# Visualize your data with dashboards
+
+OpenSearch Dashboards gives you the opportunity to explore and interpret information in an interactive visual environment. Having the right visual can help you accurately show metrics and data patterns in a clear, unbiased way.  
```

Comment:
Instead of the second sentence can we highlight that it would be impossible to see trends when you have to comb through thousands of data files without a visualization? So, something like:
Visualizations let you identify trends and spot outliers quickly for a large dataset.

Reply:
Revised